### PR TITLE
update swa 3 app

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
- add missing properties to swa 3 field 

### Resolves:
 [OKTA-147418](https://oktainc.atlassian.net/browse/OKTA-147418)

### CODE REVIEW

File: _source/_docs/api/resources/apps.md [hunk 1]
```diff
@@ -346,12 +346,13 @@ Adds a SWA application that requires a browser plugin and supports 3 CSS selecto
...
-usernameField      | CSS selector for the username field in the login form | String   | FALSE    | FALSE  |
-passwordField      | CSS selector for the password field in the login form | String   | FALSE    | FALSE  |
-buttonField        | CSS selector for the login button in the login form   | String   | FALSE    | FALSE  |
+targetURL                | The URL of the login page for this app                | String   | FALSE    | FALSE  | [URL](http://tools.ietf.org/html/rfc3986)
+usernameSelector      | CSS selector for the username field in the login form | String   | FALSE    | FALSE  |
+passwordSelector      | CSS selector for the password field in the login form | String   | FALSE    | FALSE  |
+buttonSelector        | CSS selector for the login button in the login form   | String   | FALSE    | FALSE  |
 extraFieldSelector | CSS selector for the extra field in the form          | String   | FALSE    | FALSE  |
 extraFieldValue    | Value for extra field form field                      | String   | FALSE    | FALSE  |
+loginUrlRegex     | A regular expression that further restricts `targetURL` to the specified regular expression | String | FALSE | FALSE |
```
Comment (samerfanek-okta) [inline _source/_docs/api/resources/apps.md]: Mind also adding this to the ``template_swa`` app (2 fields) after line 248?

File: _source/_docs/api/resources/apps.md [hunk 2]
```diff
@@ -372,7 +373,8 @@ curl -v -X POST \
       "userNameSelector": "#txtbox-username",
       "url": "https://example.com/login.html",
       "extraFieldSelector": ".login",
-      "extraFieldValue": "SOMEVALUE"
+      "extraFieldValue": "SOMEVALUE",
+      "loginUrlRegex": "REGEX_EXPRESSION"
```
Comment (samerfanek-okta) [inline _source/_docs/api/resources/apps.md]: * Can you also update the response example on line 424?
* Let's add the ``loginUrlRegex`` field to the 2-field SWA app request example (after line 267) and response example (after line 312)

File: _source/_docs/api/resources/apps.md (hunk 1 above)
Comment (mystiberry-okta) [reply _source/_docs/api/resources/apps.md]: done.

**Comment by mystiberry-okta** [comment]
@samerfanek-okta ready for you to review again!

File: _source/_docs/api/resources/apps.md (hunk 2 above)
Comment (mystiberry-okta) [reply _source/_docs/api/resources/apps.md]: done.

File: _source/_docs/api/resources/apps.md [hunk 3]
```diff
@@ -246,6 +246,7 @@ url           | The URL of the login page for this app                | String
 usernameField | CSS selector for the username field in the login form | String   | FALSE    | FALSE  |
 passwordField | CSS selector for the password field in the login form | String   | FALSE    | FALSE  |
 buttonField   | CSS selector for the login button in the login form   | String   | FALSE    | FALSE  |
+loginUrlRegex     | A regular expression that further restricts `targetURL` to the specified regular expression | String | FALSE | FALSE |
```
Comment (samerfanek-okta) [inline _source/_docs/api/resources/apps.md]: One final comment: This description should say `url` instead of `targetURL`

Comment (mystiberry-okta) [reply _source/_docs/api/resources/apps.md]: Fixed.

File: _source/_docs/api/resources/apps.md (hunk 1 above)
Comment (robertjd) [reply _source/_docs/api/resources/apps.md]: In this example request, we need to remove `url` and replace it with `targetURL`

**Pull request merged**